lib/node_modules/@stdlib/stats/base/dists/normal/quantile/src/main.c

```diff
@@ -20,6 +20,7 @@
 #include "stdlib/math/base/assert/is_nan.h"
 #include "stdlib/math/base/special/sqrt.h"
 #include "stdlib/math/base/special/erfinv.h"
+#include "../../../../../constants/float64/sqrt-two/include/stdlib/constants/float64/sqrt_two.h"
```

Comment:
This should not be a relative path.

```suggestion
#include "stdlib/constants/float64/sqrt_two.h"
```

The PR also needs to include this package then as a dependency in the `manifest.json` file.